composer.json

```diff
@@ -15,7 +15,8 @@
     "require": {
```

Comment:
`ext-json` is not a composer package, but it is a PHP extension which is required to have loaded for this package to function properly. Extensions can also be listed within the composer.json file